src/components/CallView/shared/LocalMediaControls.vue

```diff
@@ -590,9 +590,10 @@ export default {
 }
 
 .hint {
-	padding: 4px;
+	padding: 12px;
+	max-width: 300px;
```

Comment:
do we face the same issue with the "please use the speaker view" popover and others?